lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -11,9 +11,10 @@ module Manifest
       OAUTH_REQUIRED_FIELDS = %w[client_id client_secret authorize_uri access_token_uri].freeze
       PARAMETER_TYPES = ZendeskAppsSupport::Manifest::Parameter::TYPES
       OAUTH_MANIFEST_LINK = 'https://developer.zendesk.com/apps/docs/developer-guide/manifest#oauth'
+      SECURE_PARAM_SCOPES = %w[header body url_host url_path jwt_secret_key jwt_claim basic_auth_username basic_auth_password].freeze
 
       class << self
-        def call(package, error_on_password_parameter: false)
+        def call(package, error_on_password_parameter: false, validate_scopes_for_secure_parameter: false)
```

Comment:
Should `ZendeskAppsSupport::Package#validate` be updated to pass  `validate_scopes_for_secure_parameter` when calling this method?

https://github.com/zendesk/zendesk_apps_support/blob/897af89b49adb04172ba38ec752145a9ab6857fa/lib/zendesk_apps_support/package.rb#L41